docs/clusters/alpine/allocations.md

```diff
@@ -79,8 +81,7 @@ Step 1: Fill out the [Ascent Allocation
 Request](https://forms.office.com/r/eAA15b8Gsg) form. You need to be 
 logged in to Office365 with your CU Boulder account.
 
-Step 2: Look out for an email message from the CURC ticketing system (<rc-help@colorado.edu>) indicating when your allocation is ready to 
-use.
+Step 2: Look out for an email message from the CURC ticketing system (<rc-help@colorado.edu>) indicating when your allocation is ready to use.
 
 ### Get a Peak Allocation 
```

Comment:
Suggest changing this section header to Get a Peak Allocation (CU Boulder only)

Reply:
Suggest changing the section head above it to Get an Ascent Allocation (CU Boulder only)

Even better, we should create a section called "Allocations for CU Boulder Users" and tab the Ascent/Peak sections.

Reply:
Tabs for institutions, sections headers within with allocation type

Reply:
Remember that Trailhead is an auto-allocation. Create headers for each type (Trailhead (auto-allocation), Ascent, Peak) for each organization? Exceptions for Anschutz and CSU, with just a tab directing to the respective institutions' support contacts.